2021-03-23 09:16:59,762 DEBUG - (leriza.muralla) - Successfully loaded and updated table
2021-03-23 09:24:45,799 ERROR - Internal Server Error: /esync/euqp-proxy/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 169, in get
    context['keys'] = parse_standard_config(context['content'])
  File "/app/argus_final/app/argus/esync/lib/nginx_processor.py", line 11, in parse_standard_config
    rConfig = nginx.loads(config).as_dict
  File "/app/argus_final/venv/lib/python2.7/site-packages/nginx.py", line 402, in loads
    m = re.compile(r'^\s*events\s*{', re.S).search(data[index:])
TypeError: 'bool' object has no attribute '__getitem__'
2021-03-23 09:24:49,666 ERROR - Internal Server Error: /esync/euqp-proxy/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 169, in get
    context['keys'] = parse_standard_config(context['content'])
  File "/app/argus_final/app/argus/esync/lib/nginx_processor.py", line 11, in parse_standard_config
    rConfig = nginx.loads(config).as_dict
  File "/app/argus_final/venv/lib/python2.7/site-packages/nginx.py", line 402, in loads
    m = re.compile(r'^\s*events\s*{', re.S).search(data[index:])
TypeError: 'bool' object has no attribute '__getitem__'
2021-03-23 09:24:56,925 ERROR - Internal Server Error: /esync/euqp-proxy/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 169, in get
    context['keys'] = parse_standard_config(context['content'])
  File "/app/argus_final/app/argus/esync/lib/nginx_processor.py", line 11, in parse_standard_config
    rConfig = nginx.loads(config).as_dict
  File "/app/argus_final/venv/lib/python2.7/site-packages/nginx.py", line 402, in loads
    m = re.compile(r'^\s*events\s*{', re.S).search(data[index:])
TypeError: 'bool' object has no attribute '__getitem__'
2021-03-23 09:25:05,020 ERROR - Internal Server Error: /esync/euqp-proxy/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 169, in get
    context['keys'] = parse_standard_config(context['content'])
  File "/app/argus_final/app/argus/esync/lib/nginx_processor.py", line 11, in parse_standard_config
    rConfig = nginx.loads(config).as_dict
  File "/app/argus_final/venv/lib/python2.7/site-packages/nginx.py", line 402, in loads
    m = re.compile(r'^\s*events\s*{', re.S).search(data[index:])
TypeError: 'bool' object has no attribute '__getitem__'
2021-03-23 09:25:06,690 ERROR - Internal Server Error: /esync/euqp-proxy/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 169, in get
    context['keys'] = parse_standard_config(context['content'])
  File "/app/argus_final/app/argus/esync/lib/nginx_processor.py", line 11, in parse_standard_config
    rConfig = nginx.loads(config).as_dict
  File "/app/argus_final/venv/lib/python2.7/site-packages/nginx.py", line 402, in loads
    m = re.compile(r'^\s*events\s*{', re.S).search(data[index:])
TypeError: 'bool' object has no attribute '__getitem__'
2021-03-23 09:25:11,725 ERROR - Internal Server Error: /esync/euqp-proxy/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 169, in get
    context['keys'] = parse_standard_config(context['content'])
  File "/app/argus_final/app/argus/esync/lib/nginx_processor.py", line 11, in parse_standard_config
    rConfig = nginx.loads(config).as_dict
  File "/app/argus_final/venv/lib/python2.7/site-packages/nginx.py", line 402, in loads
    m = re.compile(r'^\s*events\s*{', re.S).search(data[index:])
TypeError: 'bool' object has no attribute '__getitem__'
2021-03-23 10:20:33,639 DEBUG - starting thread (client mode): 0xc7d0dcd0L
2021-03-23 10:20:33,639 DEBUG - Local version/idstring: SSH-2.0-paramiko_2.6.0
2021-03-23 10:20:33,658 DEBUG - Remote version/idstring: SSH-2.0-OpenSSH_7.4
2021-03-23 10:20:33,658 INFO - Connected (version 2.0, client OpenSSH_7.4)
2021-03-23 10:20:33,661 DEBUG - kex algos:[u'curve25519-sha256', u'curve25519-sha256@libssh.org', u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group16-sha512', u'diffie-hellman-group18-sha512', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha256', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'rsa-sha2-512', u'rsa-sha2-256', u'ecdsa-sha2-nistp256', u'ssh-ed25519'] client encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] server encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] client mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] server mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
2021-03-23 10:20:33,661 DEBUG - Kex agreed: curve25519-sha256@libssh.org
2021-03-23 10:20:33,661 DEBUG - HostKey agreed: ssh-ed25519
2021-03-23 10:20:33,662 DEBUG - Cipher agreed: aes128-ctr
2021-03-23 10:20:33,662 DEBUG - MAC agreed: hmac-sha2-256
2021-03-23 10:20:33,662 DEBUG - Compression agreed: none
2021-03-23 10:20:33,714 DEBUG - kex engine KexCurve25519 specified hash_algo <built-in function openssl_sha256>
2021-03-23 10:20:33,715 DEBUG - Switch to new keys ...
2021-03-23 10:20:33,740 DEBUG - Adding ssh-ed25519 host key for [154.204.50.106]:28032: 667af3f17d2850bab9c01e6da106f189
2021-03-23 10:20:33,742 DEBUG - Trying discovered key f1e2a20412f7e99b7abdb041b22f15e1 in /root/.ssh/id_rsa
2021-03-23 10:20:33,759 DEBUG - userauth is OK
2021-03-23 10:20:35,116 INFO - Authentication (publickey) successful!
2021-03-23 10:20:35,118 DEBUG - [chan 0] Max packet in: 32768 bytes
2021-03-23 10:20:35,124 DEBUG - Received global request "hostkeys-00@openssh.com"
2021-03-23 10:20:35,124 DEBUG - Rejecting "hostkeys-00@openssh.com" global request from server.
2021-03-23 10:20:35,124 DEBUG - Debug msg: Adding to environment: NAME=argus
2021-03-23 10:20:35,167 DEBUG - [chan 0] Max packet out: 32768 bytes
2021-03-23 10:20:35,167 DEBUG - Secsh channel 0 opened.
2021-03-23 10:20:35,189 DEBUG - [chan 0] Sesch channel 0 request ok
2021-03-23 10:20:35,195 DEBUG - [chan 0] Sesch channel 0 request ok
2021-03-23 10:20:35,195 DEBUG - [chan 1] Max packet in: 32768 bytes
2021-03-23 10:20:35,199 DEBUG - [chan 1] Max packet out: 32768 bytes
2021-03-23 10:20:35,199 DEBUG - Secsh channel 1 opened.
2021-03-23 10:20:35,207 DEBUG - [chan 1] Sesch channel 1 request ok
2021-03-23 10:20:35,222 DEBUG - [chan 1] EOF received (1)
2021-03-23 10:20:35,222 DEBUG - [chan 1] EOF sent (1)
2021-03-23 10:20:35,223 DEBUG - EOF in transport thread
2021-03-23 10:20:35,236 DEBUG - starting thread (client mode): 0xc779b710L
2021-03-23 10:20:35,236 DEBUG - Local version/idstring: SSH-2.0-paramiko_2.6.0
2021-03-23 10:20:35,247 DEBUG - Remote version/idstring: SSH-2.0-OpenSSH_7.4
2021-03-23 10:20:35,247 INFO - Connected (version 2.0, client OpenSSH_7.4)
2021-03-23 10:20:35,249 DEBUG - kex algos:[u'curve25519-sha256', u'curve25519-sha256@libssh.org', u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group16-sha512', u'diffie-hellman-group18-sha512', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha256', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'rsa-sha2-512', u'rsa-sha2-256', u'ecdsa-sha2-nistp256', u'ssh-ed25519'] client encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] server encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] client mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] server mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
2021-03-23 10:20:35,250 DEBUG - Kex agreed: curve25519-sha256@libssh.org
2021-03-23 10:20:35,250 DEBUG - HostKey agreed: ssh-ed25519
2021-03-23 10:20:35,250 DEBUG - Cipher agreed: aes128-ctr
2021-03-23 10:20:35,250 DEBUG - MAC agreed: hmac-sha2-256
2021-03-23 10:20:35,250 DEBUG - Compression agreed: none
2021-03-23 10:20:35,302 DEBUG - kex engine KexCurve25519 specified hash_algo <built-in function openssl_sha256>
2021-03-23 10:20:35,303 DEBUG - Switch to new keys ...
2021-03-23 10:20:35,337 DEBUG - Adding ssh-ed25519 host key for [154.204.50.106]:28032: 667af3f17d2850bab9c01e6da106f189
2021-03-23 10:20:35,338 DEBUG - Trying discovered key f1e2a20412f7e99b7abdb041b22f15e1 in /root/.ssh/id_rsa
2021-03-23 10:20:35,347 DEBUG - userauth is OK
2021-03-23 10:20:35,400 INFO - Authentication (publickey) successful!
2021-03-23 10:20:35,408 DEBUG - [chan 0] Max packet in: 32768 bytes
2021-03-23 10:20:35,411 DEBUG - Received global request "hostkeys-00@openssh.com"
2021-03-23 10:20:35,411 DEBUG - Rejecting "hostkeys-00@openssh.com" global request from server.
2021-03-23 10:20:35,411 DEBUG - Debug msg: Adding to environment: NAME=argus
2021-03-23 10:20:35,453 DEBUG - [chan 0] Max packet out: 32768 bytes
2021-03-23 10:20:35,453 DEBUG - Secsh channel 0 opened.
2021-03-23 10:20:35,478 DEBUG - [chan 0] Sesch channel 0 request ok
2021-03-23 10:20:35,483 DEBUG - [chan 0] Sesch channel 0 request ok
2021-03-23 10:20:35,484 DEBUG - [chan 1] Max packet in: 32768 bytes
2021-03-23 10:20:35,488 DEBUG - [chan 1] Max packet out: 32768 bytes
2021-03-23 10:20:35,488 DEBUG - Secsh channel 1 opened.
2021-03-23 10:20:35,495 DEBUG - [chan 1] Sesch channel 1 request ok
2021-03-23 10:20:35,514 DEBUG - [chan 1] EOF received (1)
2021-03-23 10:20:35,514 DEBUG - [chan 1] EOF sent (1)
2021-03-23 10:20:35,515 DEBUG - [chan 2] Max packet in: 32768 bytes
2021-03-23 10:20:35,548 DEBUG - [chan 2] Max packet out: 32768 bytes
2021-03-23 10:20:35,548 DEBUG - Secsh channel 2 opened.
2021-03-23 10:20:35,586 DEBUG - [chan 2] Sesch channel 2 request ok
2021-03-23 10:20:35,604 DEBUG - [chan 2] EOF received (2)
2021-03-23 10:20:35,604 DEBUG - [chan 2] EOF sent (2)
2021-03-23 10:20:35,605 DEBUG - [chan 3] Max packet in: 32768 bytes
2021-03-23 10:20:35,650 DEBUG - [chan 3] Max packet out: 32768 bytes
2021-03-23 10:20:35,650 DEBUG - Secsh channel 3 opened.
2021-03-23 10:20:35,677 DEBUG - [chan 3] Sesch channel 3 request ok
2021-03-23 10:20:35,703 DEBUG - [chan 3] EOF received (3)
2021-03-23 10:20:35,704 DEBUG - [chan 3] EOF sent (3)
2021-03-23 10:20:35,705 DEBUG - EOF in transport thread
2021-03-23 12:07:19,616 ERROR - Internal Server Error: /esync/om-fpms_pms_nginx/generate/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 287, in post
    nonstandard_esync_log(hostgroup, request.user.username, [result[1]],activity_type='CREATE')
  File "/app/argus_final/app/argus/esync/lib/esync_logger.py", line 20, in nonstandard_esync_log
    f.write(i)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 246-248: ordinal not in range(128)
2021-03-23 14:43:15,461 DEBUG - (landon) - Successfully loaded and updated table
2021-03-23 14:46:46,067 DEBUG - (landon) - Successfully loaded and updated table
2021-03-23 15:05:45,776 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2021-03-23 15:12:29,271 DEBUG - (landon) - Successfully loaded and updated table
2021-03-23 15:16:41,854 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2021-03-23 15:53:51,617 ERROR - Internal Server Error: /esync/om-fpms_pms_nginx/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 169, in get
    context['keys'] = parse_standard_config(context['content'])
  File "/app/argus_final/app/argus/esync/lib/nginx_processor.py", line 11, in parse_standard_config
    rConfig = nginx.loads(config).as_dict
  File "/app/argus_final/venv/lib/python2.7/site-packages/nginx.py", line 402, in loads
    m = re.compile(r'^\s*events\s*{', re.S).search(data[index:])
TypeError: 'bool' object has no attribute '__getitem__'
2021-03-23 17:04:29,479 ERROR - Internal Server Error: /esync/om-test_nginx/generate/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 287, in post
    nonstandard_esync_log(hostgroup, request.user.username, [result[1]],activity_type='CREATE')
  File "/app/argus_final/app/argus/esync/lib/esync_logger.py", line 20, in nonstandard_esync_log
    f.write(i)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 239-241: ordinal not in range(128)
2021-03-23 17:16:07,712 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2021-03-23 18:27:20,791 DEBUG - (landon) - Successfully loaded and updated table
2021-03-23 18:27:30,744 DEBUG - (landon) - Successfully loaded and updated table
2021-03-23 22:25:54,307 DEBUG - starting thread (client mode): 0xc0f5d1d0L
2021-03-23 22:25:54,308 DEBUG - Local version/idstring: SSH-2.0-paramiko_2.6.0
2021-03-23 22:25:54,329 DEBUG - Remote version/idstring: SSH-2.0-OpenSSH_7.4
2021-03-23 22:25:54,329 INFO - Connected (version 2.0, client OpenSSH_7.4)
2021-03-23 22:25:54,335 DEBUG - kex algos:[u'curve25519-sha256', u'curve25519-sha256@libssh.org', u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group16-sha512', u'diffie-hellman-group18-sha512', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha256', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'rsa-sha2-512', u'rsa-sha2-256', u'ecdsa-sha2-nistp256', u'ssh-ed25519'] client encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] server encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] client mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] server mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
2021-03-23 22:25:54,335 DEBUG - Kex agreed: curve25519-sha256@libssh.org
2021-03-23 22:25:54,335 DEBUG - HostKey agreed: ssh-ed25519
2021-03-23 22:25:54,335 DEBUG - Cipher agreed: aes128-ctr
2021-03-23 22:25:54,336 DEBUG - MAC agreed: hmac-sha2-256
2021-03-23 22:25:54,336 DEBUG - Compression agreed: none
2021-03-23 22:25:54,367 DEBUG - kex engine KexCurve25519 specified hash_algo <built-in function openssl_sha256>
2021-03-23 22:25:54,368 DEBUG - Switch to new keys ...
2021-03-23 22:25:54,374 DEBUG - Adding ssh-ed25519 host key for [210.209.72.125]:28032: f0e839335cba3de5a05d8dd23be98e47
2021-03-23 22:25:54,375 DEBUG - Trying discovered key f1e2a20412f7e99b7abdb041b22f15e1 in /root/.ssh/id_rsa
2021-03-23 22:25:54,413 DEBUG - userauth is OK
2021-03-23 22:25:59,454 INFO - Authentication (publickey) successful!
2021-03-23 22:25:59,477 DEBUG - [chan 0] Max packet in: 32768 bytes
2021-03-23 22:25:59,483 DEBUG - Received global request "hostkeys-00@openssh.com"
2021-03-23 22:25:59,483 DEBUG - Rejecting "hostkeys-00@openssh.com" global request from server.
2021-03-23 22:25:59,484 DEBUG - Debug msg: Adding to environment: NAME=argus
2021-03-23 22:25:59,525 DEBUG - [chan 0] Max packet out: 32768 bytes
2021-03-23 22:25:59,525 DEBUG - Secsh channel 0 opened.
2021-03-23 22:25:59,551 DEBUG - [chan 0] Sesch channel 0 request ok
2021-03-23 22:25:59,567 DEBUG - [chan 0] Sesch channel 0 request ok
2021-03-23 22:25:59,568 DEBUG - [chan 1] Max packet in: 32768 bytes
2021-03-23 22:25:59,572 DEBUG - [chan 1] Max packet out: 32768 bytes
2021-03-23 22:25:59,572 DEBUG - Secsh channel 1 opened.
2021-03-23 22:25:59,595 DEBUG - [chan 1] Sesch channel 1 request ok
2021-03-23 22:26:11,507 DEBUG - [chan 2] Max packet in: 32768 bytes
2021-03-23 22:26:11,509 DEBUG - [chan 1] EOF received (1)
2021-03-23 22:26:11,509 DEBUG - [chan 1] EOF sent (1)
2021-03-23 22:26:11,512 DEBUG - [chan 2] Max packet out: 32768 bytes
2021-03-23 22:26:11,512 DEBUG - Secsh channel 2 opened.
2021-03-23 22:26:11,564 DEBUG - [chan 2] Sesch channel 2 request ok
2021-03-23 22:26:11,585 DEBUG - [chan 2] EOF received (2)
2021-03-23 22:26:11,585 DEBUG - [chan 2] EOF sent (2)
2021-03-23 22:26:11,586 DEBUG - EOF in transport thread
2021-03-24 00:14:37,653 DEBUG - starting thread (client mode): 0xc2574210L
2021-03-24 00:14:37,654 DEBUG - Local version/idstring: SSH-2.0-paramiko_2.6.0
2021-03-24 00:14:37,950 DEBUG - Remote version/idstring: SSH-2.0-OpenSSH_6.4
2021-03-24 00:14:37,951 INFO - Connected (version 2.0, client OpenSSH_6.4)
2021-03-24 00:14:37,970 DEBUG - kex algos:[u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'ecdsa-sha2-nistp256'] client encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] server encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] client mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] server mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
2021-03-24 00:14:37,970 DEBUG - Kex agreed: ecdh-sha2-nistp256
2021-03-24 00:14:37,971 DEBUG - HostKey agreed: ecdsa-sha2-nistp256
2021-03-24 00:14:37,971 DEBUG - Cipher agreed: aes128-ctr
2021-03-24 00:14:37,971 DEBUG - MAC agreed: hmac-sha2-256
2021-03-24 00:14:37,971 DEBUG - Compression agreed: none
2021-03-24 00:14:38,512 DEBUG - kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2021-03-24 00:14:38,512 DEBUG - Switch to new keys ...
2021-03-24 00:14:38,526 DEBUG - Adding ecdsa-sha2-nistp256 host key for [103.56.16.99]:28032: 75d1c31170d6257e2001c1129928c81c
2021-03-24 00:14:38,528 DEBUG - Trying discovered key f1e2a20412f7e99b7abdb041b22f15e1 in /root/.ssh/id_rsa
2021-03-24 00:14:39,071 DEBUG - userauth is OK
2021-03-24 00:14:44,545 INFO - Authentication (publickey) successful!
2021-03-24 00:14:44,579 DEBUG - EOF in transport thread
2021-03-24 00:14:47,845 DEBUG - starting thread (client mode): 0xc4ffd090L
2021-03-24 00:14:47,845 DEBUG - Local version/idstring: SSH-2.0-paramiko_2.6.0
2021-03-24 00:14:48,138 DEBUG - Remote version/idstring: SSH-2.0-OpenSSH_6.4
2021-03-24 00:14:48,138 INFO - Connected (version 2.0, client OpenSSH_6.4)
2021-03-24 00:14:48,397 DEBUG - kex algos:[u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'ecdsa-sha2-nistp256'] client encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] server encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] client mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] server mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
2021-03-24 00:14:48,397 DEBUG - Kex agreed: ecdh-sha2-nistp256
2021-03-24 00:14:48,397 DEBUG - HostKey agreed: ecdsa-sha2-nistp256
2021-03-24 00:14:48,397 DEBUG - Cipher agreed: aes128-ctr
2021-03-24 00:14:48,397 DEBUG - MAC agreed: hmac-sha2-256
2021-03-24 00:14:48,397 DEBUG - Compression agreed: none
2021-03-24 00:14:48,700 DEBUG - kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2021-03-24 00:14:48,701 DEBUG - Switch to new keys ...
2021-03-24 00:14:48,717 DEBUG - Adding ecdsa-sha2-nistp256 host key for [103.56.16.99]:28032: 75d1c31170d6257e2001c1129928c81c
2021-03-24 00:14:48,718 DEBUG - Trying discovered key f1e2a20412f7e99b7abdb041b22f15e1 in /root/.ssh/id_rsa
2021-03-24 00:14:49,257 DEBUG - userauth is OK
2021-03-24 00:14:54,726 INFO - Authentication (publickey) successful!
2021-03-24 00:14:54,728 DEBUG - [chan 0] Max packet in: 32768 bytes
2021-03-24 00:14:55,140 DEBUG - Debug msg: Adding to environment: NAME=argus
2021-03-24 00:14:55,438 DEBUG - [chan 0] Max packet out: 32768 bytes
2021-03-24 00:14:55,438 DEBUG - Secsh channel 0 opened.
2021-03-24 00:14:55,709 DEBUG - [chan 0] Sesch channel 0 request ok
2021-03-24 00:14:55,974 DEBUG - [chan 0] Sesch channel 0 request ok
2021-03-24 00:14:55,975 DEBUG - [chan 1] Max packet in: 32768 bytes
2021-03-24 00:14:56,236 DEBUG - [chan 1] Max packet out: 32768 bytes
2021-03-24 00:14:56,237 DEBUG - Secsh channel 1 opened.
2021-03-24 00:14:56,504 DEBUG - [chan 1] Sesch channel 1 request ok
2021-03-24 00:15:26,547 DEBUG - [chan 2] Max packet in: 32768 bytes
2021-03-24 00:15:26,549 DEBUG - [chan 1] EOF received (1)
2021-03-24 00:15:26,549 DEBUG - [chan 1] EOF sent (1)
2021-03-24 00:15:26,808 DEBUG - [chan 2] Max packet out: 32768 bytes
2021-03-24 00:15:26,809 DEBUG - Secsh channel 2 opened.
2021-03-24 00:15:27,379 DEBUG - [chan 2] Sesch channel 2 request ok
2021-03-24 00:15:27,598 DEBUG - [chan 2] EOF received (2)
2021-03-24 00:15:27,599 DEBUG - [chan 2] EOF sent (2)
2021-03-24 00:15:27,600 DEBUG - EOF in transport thread
2021-03-24 00:22:29,458 DEBUG - starting thread (client mode): 0xc6690510L
2021-03-24 00:22:29,459 DEBUG - Local version/idstring: SSH-2.0-paramiko_2.6.0
2021-03-24 00:22:29,762 DEBUG - Remote version/idstring: SSH-2.0-OpenSSH_6.4
2021-03-24 00:22:29,763 INFO - Connected (version 2.0, client OpenSSH_6.4)
2021-03-24 00:22:29,788 DEBUG - kex algos:[u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'ecdsa-sha2-nistp256'] client encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] server encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] client mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] server mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
2021-03-24 00:22:29,789 DEBUG - Kex agreed: ecdh-sha2-nistp256
2021-03-24 00:22:29,789 DEBUG - HostKey agreed: ecdsa-sha2-nistp256
2021-03-24 00:22:29,789 DEBUG - Cipher agreed: aes128-ctr
2021-03-24 00:22:29,789 DEBUG - MAC agreed: hmac-sha2-256
2021-03-24 00:22:29,789 DEBUG - Compression agreed: none
2021-03-24 00:22:30,284 DEBUG - kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2021-03-24 00:22:30,285 DEBUG - Switch to new keys ...
2021-03-24 00:22:30,299 DEBUG - Adding ecdsa-sha2-nistp256 host key for [103.56.16.109]:28032: 75d1c31170d6257e2001c1129928c81c
2021-03-24 00:22:30,300 DEBUG - Trying discovered key f1e2a20412f7e99b7abdb041b22f15e1 in /root/.ssh/id_rsa
2021-03-24 00:22:30,843 DEBUG - userauth is OK
2021-03-24 00:22:36,361 INFO - Authentication (publickey) successful!
2021-03-24 00:22:36,369 DEBUG - EOF in transport thread
2021-03-24 00:22:39,630 DEBUG - starting thread (client mode): 0xc3b2c750L
2021-03-24 00:22:39,630 DEBUG - Local version/idstring: SSH-2.0-paramiko_2.6.0
2021-03-24 00:22:39,948 DEBUG - Remote version/idstring: SSH-2.0-OpenSSH_6.4
2021-03-24 00:22:39,948 INFO - Connected (version 2.0, client OpenSSH_6.4)
2021-03-24 00:22:40,201 DEBUG - kex algos:[u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'ecdsa-sha2-nistp256'] client encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] server encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] client mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] server mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
2021-03-24 00:22:40,202 DEBUG - Kex agreed: ecdh-sha2-nistp256
2021-03-24 00:22:40,202 DEBUG - HostKey agreed: ecdsa-sha2-nistp256
2021-03-24 00:22:40,202 DEBUG - Cipher agreed: aes128-ctr
2021-03-24 00:22:40,202 DEBUG - MAC agreed: hmac-sha2-256
2021-03-24 00:22:40,202 DEBUG - Compression agreed: none
2021-03-24 00:22:40,459 DEBUG - kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2021-03-24 00:22:40,460 DEBUG - Switch to new keys ...
2021-03-24 00:22:40,469 DEBUG - Adding ecdsa-sha2-nistp256 host key for [103.56.16.109]:28032: 75d1c31170d6257e2001c1129928c81c
2021-03-24 00:22:40,470 DEBUG - Trying discovered key f1e2a20412f7e99b7abdb041b22f15e1 in /root/.ssh/id_rsa
2021-03-24 00:22:41,007 DEBUG - userauth is OK
2021-03-24 00:22:46,475 INFO - Authentication (publickey) successful!
2021-03-24 00:22:46,477 DEBUG - [chan 0] Max packet in: 32768 bytes
2021-03-24 00:22:46,731 DEBUG - Debug msg: Adding to environment: NAME=argus
2021-03-24 00:22:47,024 DEBUG - [chan 0] Max packet out: 32768 bytes
2021-03-24 00:22:47,024 DEBUG - Secsh channel 0 opened.
2021-03-24 00:22:47,310 DEBUG - [chan 0] Sesch channel 0 request ok
2021-03-24 00:22:47,570 DEBUG - [chan 0] Sesch channel 0 request ok
2021-03-24 00:22:47,571 DEBUG - [chan 1] Max packet in: 32768 bytes
2021-03-24 00:22:47,828 DEBUG - [chan 1] Max packet out: 32768 bytes
2021-03-24 00:22:47,828 DEBUG - Secsh channel 1 opened.
2021-03-24 00:22:48,102 DEBUG - [chan 1] Sesch channel 1 request ok
2021-03-24 00:23:18,180 DEBUG - [chan 0] EOF sent (0)
2021-03-24 00:23:18,182 DEBUG - [chan 1] EOF sent (1)
2021-03-24 00:23:18,184 DEBUG - EOF in transport thread
